### PR TITLE
Add Indonesian language for WebUI

### PR DESCRIPTION
**Pull request by bpanca05**
## Changes

Add Indonesian language for WebUI

## Checkmarks

- [x] The modified functions have been tested.
- [x] Used the same indentation as the rest of the project.
- [x] Updated documentation (changelog).

### CODE REVIEW

File: TRANSLATOR.md
Comment (RainyXeon) [inline TRANSLATOR.md:1]: Can you please make this list synchronized alphabetically?

**Review by RainyXeon — COMMENTED** [review]
Check review below

File: TRANSLATOR.md
Comment (bpanca05) [reply TRANSLATOR.md:1]: Done ✅

**Comment by RainyXeon** [comment]
LGTM

**Pull request merged**